Support certificates configuration in TLSStore CRD (#8976)
traefik/traefik: docs/content/https/tls.md

### Strict SNI Checking

With strict SNI checking enabled, Traefik won't allow connections from clients that do not specify a server_name extension 

> or don't match any configured certificate (which does not include the DefaultCertificate).

Reviewer: ```suggestion
or don't match any of the configured certificates. The DefaultCertificate is irrelevant on that matter.
```
